src/query/graphite/graphite/timespec.go

```diff
@@ -108,9 +137,146 @@ func ParseTime(s string, now time.Time, absoluteOffset time.Duration) (time.Time
 		return time.Unix(n, 0).UTC(), nil
 	}
 
+	ref, offset := s, ""
+	if strings.Contains(s, "+") {
+		stringSlice := strings.Split(s, "+")
+		ref, offset = stringSlice[0], stringSlice[1]
+		offset = "+" + offset
+	} else if strings.Contains(s, "-") {
+		stringSlice := strings.Split(s, "-")
+		ref, offset = stringSlice[0], stringSlice[1]
+		offset = "-" + offset
+	}
+	err = nil
```

Comment:
No need to make err = nil, it will be shadowed/rewritten below, it's not required to reset it here.

Reply:
Got it

Reply:
OK, good to know. Thanks.